README.rst

```diff
@@ -95,6 +96,9 @@ Please read `Get Started <https://dvc.org/doc/get-started>`_ guide for a full ve
 |                                   | | ``$ dvc push``                                                           |
 +-----------------------------------+----------------------------------------------------------------------------+
 
+Prefer to work in your IDE? Check out the [DVC Extension for Visual Studio
```

Comment:
doesn't look very natural tbh

Reply:
sorry, I meant that the whole paragraph feels a bit too much here - looks like an ad a bit. If we have enough organic mentions, I would say we can completely skip it, wdyt?